src/main/resources/META-INF/plugin.xml

```diff
@@ -10,7 +10,6 @@
     ]]></description>
 
 
-    <depends>com.intellij.modules.platform</depends>
```

Comment:
Was it redundant or incorrect? :thinking:

Reply:
And how about 2019.2/2019.3, also can be/should be removed wrt. those versions?

Reply:
We no longer depend on it

Reply:
`no longer`... as in, there used to be a time when we did? :thinking:

Reply:
Whoops, it is required indeed. THANKS for pointing this out.

Refs:
https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_configuration_file.html
https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html#modules-available-in-all-products